CONTRIBUTING.md

```diff
@@ -10,6 +10,33 @@ Use HashiCorp's [Plugin Development](https://www.terraform.io/plugin) guide as a
 
 See the [Developing The Provider](#developing-the-provider) section below for specifics about this GitLab provider.
 
+### GitPod Environment Setup
+
+You can choose to use your own development environment if desired, however a `.gitpod.yml` file is included within the repository to allow the use of GitPod easily. 
+This will allow you to use GitPod's integration with GitHub to instantly stand up a web-based development environment including Go and Docker, which are necessary 
```

Comment:
```suggestion
This will allow you to use GitPod's integration with GitHub to instantly start up a web-based development environment including Go and Docker, which are necessary 
```